releng/plugins/org.polarsys.kitalpha.releng.targets/kitalpha.targetplatform

```diff
@@ -83,9 +83,13 @@ location eclipse-release "https://download.eclipse.org/releases/2019-06/" {
 	org.eclipse.emf.ecoretools.sdk.feature.group lazy
 	org.eclipse.emf.ecoretools.design.feature.group lazy
 	org.eclipse.emf.sdk.feature.group lazy
+	org.eclipse.acceleo.doc lazy
+	org.eclipse.acceleo.source.feature.group lazy
+	org.eclipse.acceleo.feature.group lazy
+	org.eclipse.emf.cdo.feature.group lazy
 }
 
-location sirius "https://download.eclipse.org/sirius/updates/releases/6.3.0/2019-06/" {
+location sirius "https://download.eclipse.org/sirius/updates/releases/6.3.2/2019-06/" {
```

Comment:
It should be https://download.eclipse.org/sirius/updates/stable/6.4.0-S20200716-044517/2020-06/ in the meantime I think

Reply:
You are right

Reply:
Done, build in progress